src/pages/PullRequestPage/PullCoverage/routes/IndirectChangesTab/FileDiff/FileDiff.test.jsx

```diff
@@ -17,9 +20,57 @@ vi.mock('ui/CodeRenderer/hooks/useScrollToLine', async () => {
     useScrollToLine: mocks.useScrollToLine,
   }
 })
-window.requestAnimationFrame = (cb) => cb()
+
+vi.mock('shared/featureFlags', () => ({
+  useFlags: mocks.useFlags,
+}))
+
+vi.mock('@sentry/react', () => {
+  const originalModule = vi.importActual('@sentry/react')
+  return {
+    ...originalModule,
+    withProfiler: mocks.withProfiler,
+    captureMessage: mocks.captureMessage,
+  }
+})
+
+window.requestAnimationFrame = (cb) => {
+  cb(1)
+  return 1
+}
 window.cancelAnimationFrame = () => {}
 
+const scrollToMock = vi.fn()
+window.scrollTo = scrollToMock
+window.scrollY = 100
+
+class ResizeObserverMock {
```

Comment:
Is this class being used for something?

Reply:
It's being used in the underlying virtual diff renderer for a couple of things.